lib/core-integration/src/Test/Integration/Scenario/CLI/Shelley/Transactions.hs

```diff
@@ -418,7 +492,8 @@ spec = describe "SHELLEY_CLI_TRANSACTIONS" $ do
                 it title $ \ctx -> runResourceT $ do
                     wallet <- emptyWallet ctx
                     (Exit code, Stdout out, Stderr err) <-
-                        listTransactionsViaCLI ctx $ join
+                        listTransactionsViaCLI ctx TxMetadataDetailedSchema $
+                        join
```

Comment:
Indentation of `join` is misleading — it's an argument to `listTransactionsViaCLI` and merits further indentation. The position on the previous line was better in that regard.